### PR TITLE
This is the desktop version of my portfolio

### PR DESCRIPTION
**Pull request by Dinma-Faith-zz**
In this milestone  :+1: 
- I used HTML and CSS
- Added new images in it.

Comments and suggestions are welcomed
Thanks

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,17 +18,24 @@
         <div class="header">
           <div class="logo"><a href="#">My Logo</a></div>
           <nav>
+            <div class="menu-content">
```
Comment (thecodechaser) [inline index.html]: - [ ] You did a great job on the styles but for this project, you are supposed to stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1 )template as much as possible. Please place some left, right margins on the header and use the correct color for the logo to make sure it matches the design on the template.

### Your design

image

### Figma design

image

File: index.html [hunk 2]
```diff
@@ -18,17 +18,24 @@
...
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+              </ul>
+            </div>
             <span class="bars"><i class="fa-solid fa-bars"></i></span>
           </nav>
         </div>
       </header>
 
       <section class="intro">
```
Comment (thecodechaser) [inline index.html]: - [ ] Please reduce the gap between social media icons and make sure it matches the design on the template.

### Your design

image

### Figma design

image

File: index.html [hunk 3]
```diff
@@ -63,133 +70,158 @@ <h4 class="connect">LET'S CONNECT</h4>
...
+            <div class="projects">
+              <ul>
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            
+            <div>
+              <button type="submit" class="btn">See Project</button>
```
Comment (thecodechaser) [inline index.html:125]: - [ ] Please use the correct border color for all of your buttons to match the design on the template.

### Your design

image

### Figma design

image

File: index.html [hunk 4]
```diff
@@ -198,14 +230,14 @@ <h2 class="h2">Multi-Post<br />Stories</h2>
     <section class="container-section2">
```
Comment (thecodechaser) [inline index.html]: - [ ] Please use the correct right margin for this section to match the Figma design.

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Dinma-Faith👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 5]
```diff
@@ -349,3 +363,250 @@ input.form-control::placeholder {
...
+  }
+
+  .socials {
+    display: flex;
+    justify-content: space-between;
+    width: 13%;
+    padding-top: 15px;
+    margin-bottom: 100px;
+    margin-left: 123px;
+  }
```
Comment (AmaduKamara) [inline style.css]: - [ ] Kindly adjust the spacing between each social media icon below as requested by the previous reviewer.
- Currently it's not displaying at around 768px - 1000px view. Please fix that 👍

** Your Design **
image

** The Figma Design **
image

File: style.css [hunk 6]
```diff
@@ -349,3 +363,250 @@ input.form-control::placeholder {
...
+    padding-top: 142px;
+    margin: 100px auto;
+    width: 100%;
+  }
+
+  .intro2 {
+    margin-left: 80px;
+    width: 400px;
+    margin-top: -100px;
+  }
```
Comment (AmaduKamara) [inline style.css]: - [ ] Please fix the width of these sections to have `40%` on the left and `60%` on the right in order to match the design as previously asked by the first reviewer. 

** Your design **
image

** The Figma Design **
image

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️

Hi @Dinma-Faith,

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Suggested changes
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by OLIPLICHE — APPROVED** [review]
Hi @Dinma-Faith, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
- Please, go through your design again and make sure it looks like the one in Figma. This will make your portfolio more attractive and it might matter for future recruiters.

- The picture below is from Figma design. I drew the red lines to get your focus on this section.
kindly compare with your design from your browser and solve the issues. 
 
d1

  
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**